engine/js/BertaGalleryRow.js

```diff
@@ -198,114 +170,88 @@ var BertaGalleryRow = new Class({
     liElement.addClass('selected');
   },
 
-  load: function (src, mType, mWidth, mHeight, videoPath, autoPlay, caption, bDeleteExisting, xImgIndex, srcset) {
-    switch (this.phase) {
-      case 'fadeout':
-        this.imageFadeOutFx.cancel();
-        break;
-      case 'fadein':
-        this.imageResizeFx.cancel();
-        this.imageShowFx.cancel();
-        break;
-      default:
-        this.imageShowFx.cancel();
-        break;
-    }
-
+  load: function (src, mType, mWidth, mHeight, videoPath, autoPlay, caption, xImgIndex, srcset) {
+    this.currentItem += 1;
+    this.itemIsLoaded = this.currentItem <= this.loadedItems;
     this.currentSrc = null;
-    this.load_Render(src, mType, mWidth, mHeight, videoPath, autoPlay, caption, bDeleteExisting, xImgIndex, srcset);
+    this.load_Render(src, mType, mWidth, mHeight, videoPath, autoPlay, caption, xImgIndex, srcset);
```

Comment:
I see that there are a lot of parameters to this function, but maybe we should pass the `isLoaded` parameter as well? I think it could make the code a bit easier to follow. And it doesn't seem like, it's used in lot of functions around.